### DISCUSSION TITLE
Install to specific directory

### DISCUSSION THREAD

**Discussion by jh197033** [Intermesh/groupoffice#929] [Ideas]
Is it possible when I use apt-get install groupoffice to add the ability for the user to specify where the root of GO will be installed, where the tmp and file_storage will be stored on the install page?

I am trying to use Virtualmin to host the email and then install GO to connect to their mail server and want to install GO into VirtualMin apache2 root 

Install website to:
/home/groupoffice/public_html

file_storage:
/home/groupoffice/file_storage

tmp:
/home/groupoffice/tmp

Email from Virtualmin:
/home/groupoffice/MailDir etc......

Is it possibile?

Regards

James

**Comment by mschering** [comment]
No that is not possible with debian packages. Then you will have to use the tarball install method.

**Reply by jh197033** [reply]
I tried that earlier today today, couldnt use the postfix admin or IMAP AUTH modules in GO to talk to the Virtualmin postfix email accounts. I could only get it to see the email accounts if I manually entered mail server IP, IMAP account username and password.

Nevermind, it was just a thought that maybe it was possible.

I have one more question regarding the mailserver in GO
apt-get install groupoffice groupoffice-mailserver

Is the mail server production ready that I can install groupoffice and groupoffice-mailserver on debian or ubuntu as per:
https://groupoffice.readthedocs.io/en/latest/install/debian.html#mailserver

I worry I will get hit by spammers etc... is it ok to use out of the box in production? (Sorry for the extra question but really love groupoffice and want to use it)

Regards

James

**Reply by mschering** [reply]
Yes it's ready for production. By default, accounts can't be used externally unless you specifically enable it in the account properties. Also install fail2ban for extra security.

**Reply by mschering** [reply]
And external clients can send a maximum of 2 messages per minute by default.
